AutocompleteClient/FW/Logic/Request/CIOSearchQuery.swift

```diff
@@ -72,14 +78,15 @@ public struct CIOSearchQuery: CIORequestData {
      let searchQuery = CIOSearchQuery(query: "red", filters: CIOQueryFilters(groupFilter: nil, facetFilters: facetFilters), page: 1, perPage: 30, section: "Products", hiddenFields: ["price_CA", "currency_CA"])
```

Comment:
Forgot to mention this in the previous review, but could we also update the usage examples as well? Thanks!